google-cloud-tools-plugin/src/com/google/gct/idea/elysium/GoogleUserModelItem.java

```diff
@@ -98,7 +106,7 @@ public void synchronize() {
       @Override
       public void run() {
         try {
-          loadUserProjects(myUser);
```

Comment:
Notice that loadUserProjects is called with myUser, and then in the method, the user argument is only used to create the Cloud Resource Manager client.

I relocated building the client to the constructor because I wanted to unit test this method, but couldn't find a way to get the projects in GoogleUserModelItem. I still think it's a good idea to cleanup the code, but I'm not sure if passing that argument like that achieves anything. I don't think so, it should be safe to remove the argument, but maybe you can double check this.